assets/styles/mobile.css

```diff
@@ -0,0 +1,360 @@
+    max-width: 900px;
+    height: 650px;
+    padding: 10px;
+    gap: 6px;
+  }
+
+  .podium {
+    display: flex;
+    align-items: center;
+    margin-right: auto;
+    height: 100px;
+    background-image:
+      radial-gradient(
+        circle at 99% 33%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 37%,
+        transparent 37%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 46% 16%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 43%,
+        transparent 43%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 99% 25%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 22%,
+        transparent 22%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 57% 88%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 86%,
+        transparent 86%,
+        transparent 100%
+      ),
+      radial-gradient(
+        circle at 82% 78%,
+        hsl(235deg 0% 48% / 3%) 0%,
+        hsl(235deg 0% 48% / 3%) 22%,
+        transparent 22%,
+        transparent 100%
+      ),
+      linear-gradient(
+        90deg,
+        rgb(40 37 37),
+        rgb(40 37 37)
+      );
+  }
+
+  .podium > p {
+    text-align: start;
+    font-size: 3vw;
+    width: 70%;
+    color: #fdfdf7;
+    margin-left: auto;
+  }
+
+  .podium > h2 {
+    color: #ec5242;
+    font-size: 18px;
+    padding: 7px;
+  }
+
+  .all-programs {
+    height: 100px;
+    padding: 0;
+  }
+
+  #join {
+    box-sizing: border-box;
+    background-color: #f72509d2;
+    border-radius: 4px;
+    padding: 25px;
+    max-width: 320px;
+    margin: 20px auto;
+    text-align: center;
+    font-weight: 700;
+  }
+
+  .bottom-title {
+    display: none;
+  }
+
+  #program-footer {
+    text-transform: uppercase;
+    text-decoration: none;
+    box-sizing: border-box;
+    padding: 30px;
+    background-color: #df210c;
+    margin-left: -50%;
+    margin-top: -20%;
+  }
+
+  .speaker-container {
+    max-width: 500px;
+    height: 100%;
+    gap: 20px;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .speaker-heading {
+    margin-left: 100px;
+    padding: 20px;
+    font-size: 2.5vw;
+    align-items: center;
+  }
+
+  .speaker-heading > h3 {
+    text-align: center;
+  }
+
+  .speaker-text {
+    font-size: 10px;
+  }
+
+  .name,
+  .para {
+    color: #ec5242;
+    font-size: 1.7vw;
+  }
+
+  .see-more-btn {
+    padding: 12px 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
+    width: 100%;
+    border: 1px solid #d3d3d3;
+    margin-top: 10px;
+    font-size: 15px;
+    font-weight: 700;
+    text-transform: uppercase;
+    color: #272a31;
+  }
+
+  .fa-chevron-down {
+    color: #ec5242;
+  }
+
+  .top-footer {
+    display: none;
+  }
+
+  .bottom-footer {
+    background-color: #272a31;
+  }
```

Comment:
[oPtiONAL] please notice there is no footer in the mobile home page in the original design currently your mobile home page includes the footer :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210616311-56de0a02-0311-4e31-bbaf-484c50ea62a2.png)

how the home page should be according to the design guidelines please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210616490-6e20a496-52fe-4247-a55c-36bcd794b226.png)

Reply:
- Thanks for this I decided to remove the footer so that I can adhere to the design provided

-  
![Screenshot from 2023-01-04 21-51-21](https://user-images.githubusercontent.com/55842414/210628187-bce821b9-f148-433e-9332-8fc714bd8e3b.png)